### PR TITLE
[READY TO MERGE] French translation

### PR DESCRIPTION
**Pull request by itskatt**
French translation for the game.

### CODE REVIEW

**Comment by JustYanns** [comment]
Je chipote mais je tiens à te féliciter pour la quantité de travail que tu as fourni ! 😉

**Comment by itskatt** [comment]
@JustYanns Merci beaucoup !
C'est vrai, les erreurs se glissent facilement quand il y a beaucoup a traduire, donc encore merci de prendre le temps de me relire ! 
En ce qui concerne le choix d'avoir capitalisé les noms des ressources/bâtiment, c'est parce que c'est le cas dans le [fichier original](https://github.com/Anuken/Mindustry/blob/master/core/assets/bundles/bundle.properties#L765), je voulais juste y rester consistent.

**Comment by JustYanns** [comment]
En effet j'ai fait l'expérience de nombreuses faute (ayant traduit le bundle_fr il y a longtemps) ... Mais le fichier avait été presque entièrement modifié depuis à cause des mises à jour ^^

Il est vrai que je n'ai pas

**Comment by JustYanns** [comment]
There are 2 other pull requests about bundle.fr, you should tell the others that your translation is way more complete 😉

**Comment by itskatt** [comment]
Yes, I've already done it for #615, #604 and #599 !

**Comment by Bryggel** [comment]
No @itskatt some words are delete or untranslated i make a huge work, wait aging :)

**Comment by itskatt** [comment]
@Bryggel Can you please explain why big chunks (for ex the tutorial) of your translation are identical to mine? I am highly suspecting you of stealing my work.

**Comment by Bryggel** [comment]
@itskatt Its a fusion not a theft, your files are not copyright so free use sorry
If we reason like that we can do nothing, I merged mine with yours cuz nothing was perfect. The copy paste argument in this project does not hold. When I use the printf function, I steal work from other devs ? When I use open source software and modify it, I steal it? No since it's free, your translation was free and open to all that's what you do not understand.
Sorry i believe that Anuke dont give a shit

**Comment by Bryggel** [comment]
Copy and  modify a file is not stealing, fusion, i have mentioned you and your work #615

**Comment by itskatt** [comment]
@Bryggel
I understand that you are trying to help, but taking work from one pr to yours **is** what I call stealing. Also, [you are taking full credit](https://github.com/Anuken/Mindustry/commit/e17cc174080b0f85a179c50fef61ed21aef3d863) from it!

File: core/assets/bundles/bundle_fr.properties [hunk 1]
```diff
@@ -206,76 +206,78 @@ waves.copy = Copier dans le Presse-papiers
...
-toolmode.replace = Replace
-toolmode.replace.description = Draws only on solid blocks.
-toolmode.replaceall = Replace All
-toolmode.replaceall.description = Replace all blocks in map.
+editor.selectmap = Sélectionnez une carte:
+
+toolmode.replace = Remplacer
+toolmode.replace.description = Dessiner seulement sur les blocs solides.
+toolmode.replaceall = Tout remplacer
+toolmode.replaceall.description = Remplace tout les blocs de la carte.
```
Comment (AzariasB) [inline core/assets/bundles/bundle_fr.properties]: tous les blocs

File: core/assets/bundles/bundle_fr.properties [hunk 2]
```diff
@@ -288,32 +290,34 @@ filter.option.falloff = Diminution
...
 settings = Paramètres
 tutorial = Tutoriel
 tutorial.retake = Re-Take Tutorial
 editor = Éditeur
 mapeditor = Éditeur de carte
 donate = Faire un\ndon
+
 abandon = Abandon
-abandon.text = Cette zone et tous ses ressources vont être perdues.
+abandon.text = Cette zone et toute ses ressources vont être perdues.
```
Comment (AzariasB) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
abandon.text = Cette zone et toutes ses ressources vont être perdues.
```

File: core/assets/bundles/bundle_fr.properties [hunk 3]
```diff
@@ -322,69 +326,73 @@ bestwave = [LIGHT_GRAY]Meilleur: {0}
...
-configure.locked = [LIGHT_GRAY]Atteigner la vague {0}\npour configurer le transfert des ressources
+configure.locked = [LIGHT_GRAY]Atteignez la vague {0}\npour configurer le transfert des ressources
 zone.unlocked = [LIGHT_GRAY]{0} Débloquée.
 zone.requirement.complete = Vague {0} atteinte:\n{1} Exigences de la zone complétées
 zone.config.complete = Vague {0} atteinte:\nConfiguration du transfert débloquée.
 zone.resources = Ressources détectées:
 zone.objective = [lightgray]Objective: [accent]{0}
 zone.objective.survival = Survive
-zone.objective.attack = Destroy Enemy Core
+zone.objective.attack = Détruire le noyau ennemie
```
Comment (AzariasB) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
zone.objective.attack = Détruire le noyau ennemi
```

File: core/assets/bundles/bundle_fr.properties [hunk 4]
```diff
@@ -322,69 +326,73 @@ bestwave = [LIGHT_GRAY]Meilleur: {0}
...
 zone.nuclearComplex.name = Complexe nucléaire
-zone.overgrowth.name = Overgrowth
-zone.tarFields.name = Tar Fields
-zone.saltFlats.name = Salt Flats
+zone.overgrowth.name = Surcroissance
+zone.tarFields.name = Champs de Pétrole
+zone.saltFlats.name = Marais Salants
 zone.impact0078.name = Impact 0078
-zone.crags.name = Crags
+zone.crags.name = Rochets
```
Comment (AzariasB) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
zone.crags.name = Rochers
```

File: core/assets/bundles/bundle_fr.properties [hunk 5]
```diff
@@ -393,14 +401,14 @@ no = Non
...
-attackpvponly = [scarlet]Only available in Attack/PvP modes
+attackpvponly = [scarlet]Seulement disponible dans les modes Attaque et PvP
 blocks.input = Input
 blocks.output = Output
 blocks.booster = Booster
 block.unknown = [LIGHT_GRAY]???
 blocks.powercapacity = Capacité d'énergie
 blocks.powershot = Énergie/Tir
-blocks.damage = Damage
+blocks.damage = Dégats
```
Comment (AzariasB) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
blocks.damage = Dégâts
```

File: core/assets/bundles/bundle_fr.properties [hunk 6]
```diff
@@ -415,41 +423,44 @@ blocks.itemcapacity = Stockage
...
 blocks.maxunits = Nombre d'unités actives maximal
 blocks.health = Santé
-blocks.buildtime = Build Time
+blocks.buildtime = Temps de construction
 blocks.inaccuracy = Précision
 blocks.shots = Tir
 blocks.reload = Tirs/Seconde
 blocks.ammo = Munitions
+
 bar.drilltierreq = Better Drill Required
```
Comment (AzariasB) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
 bar.drilltierreq = Meilleure foreuse requise
```

File: core/assets/bundles/bundle_fr.properties [hunk 7]
```diff
@@ -471,35 +482,35 @@ category.shooting = Défense
...
+setting.difficulty.insane = Extrême
 setting.difficulty.name = Difficulté:
 setting.screenshake.name = Tremblement de l'écran
 setting.effects.name = Montrer les effets
 setting.sensitivity.name = Sensibilité de la manette
 setting.saveinterval.name = Intervalle des sauvegardes auto
 setting.seconds = {0} secondes
 setting.fullscreen.name = Plein écran
-setting.borderlesswindow.name = Borderless Window[LIGHT_GRAY] (may require restart)
+setting.borderlesswindow.name = Ecran sans bords[LIGHT_GRAY] (peut requerir le redémarrage du jeu)
```
Comment (AzariasB) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
setting.borderlesswindow.name = Ecran sans bords[LIGHT_GRAY] (peut requérir le redémarrage du jeu)
```

File: core/assets/bundles/bundle_fr.properties [hunk 8]
```diff
@@ -508,12 +519,12 @@ setting.sfxvol.name = Volume des SFX
...
 setting.chatopacity.name = Opacité du chat
-setting.playerchat.name = Display In-Game Chat
-uiscale.reset = UI scale has been changed.\nPress "OK" to confirm this scale.\n[scarlet]Reverting and exiting in[accent] {0}[] settings...
-uiscale.cancel = Cancel & Exit
-setting.bloom.name = Bloom
-keybind.title = Paramétrer les touches
-keybinds.mobile = [scarlet]Most keybinds here are not functional on mobile. Only basic movement is supported.
+setting.playerchat.name = Montrer le chat
+uiscale.reset = L'échelle de l'UI a été modifiée.\nAppuyez sur "OK" pour confirmer.\n[scarlet]Rétablissement aux parametres d'avant et fermeture dans [accent] {0}[]...
+uiscale.cancel = Anuler & Quitter
```
Comment (AzariasB) [inline core/assets/bundles/bundle_fr.properties]:
```suggestion
uiscale.cancel = Annuler & Quitter
```

**Comment by itskatt** [comment]
@AzariasB Merci pour les corrections!

**Pull request merged**